dockers/docker-snmp-sv2/start.sh

```diff
@@ -9,6 +9,15 @@ sonic-cfggen -d -y /etc/sonic/snmp.yml -t /usr/share/sonic/templates/snmpd.conf.
 mkdir -p /var/sonic
 echo "# Config files managed by sonic-config-engine" > /var/sonic/config_status
 
+CURRENT_HOSTNAME=`hostname`
+HOSTNAME=`sonic-cfggen -d -v DEVICE_METADATA[\'localhost\'][\'hostname\']`
+
+echo $HOSTNAME > /etc/hostname
+hostname -F /etc/hostname
```

Comment:
why not use `hostname $HOSTNAME`?

Reply:
> why not use `hostname $HOSTNAME`?

In this case /etc/hostname will contain the old hostname.

Reply:
@qiluo-msft  Or you meant only this line: "hostname -F /etc/hostname"? If yes, this can be changed.

Reply:
I see. Let's keep as is.

---
In reply to: [274999450](https://github.com/Azure/sonic-buildimage/pull/2773#discussion_r274999450) [](ancestors = 274999450)